integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -313,6 +313,8 @@ fuse:
 #     alluxio-hdfs-config: hdfsConfig
 #   worker: # Shared by worker and jobWorker containers
 #     alluxio-hdfs-config: hdfsConfig
+#   logserver: # Used by the logserver container
+#     alluxio-hdfs-config: hdfsConfig
```

Comment:
Not sure if there's a better example

Reply:
logserver doesn't need to talk to hdfs so the hdfs secret is not required, but his placeholder is fine